.github/workflows/autoupdate.yml

```
@@ -116,19 +116,19 @@ jobs:
             # We cannot use the -A option of `gh pr list` because the PR author changes with the Personal Access Token used,
             # so we filter on the 3rd output field (headRefName, i.e. remote_repository:branch).
             if gh_pr_list_out=$(gh pr list --search "head:$REPO" | grep -P "^\d+\t[^\t]+\tplanemo-autoupdate:$REPO\t"); then
-              OLD_TITLE=$(echo "$gh_pr_list_out" | cut -f 2)
+              OLD_TITLE=$(echo "$gh_pr_list_out" | cut -f 2 | head -n 1)
```

I don't GitHub lets you open more than one pull request from the same branch.

```suggestion
              OLD_TITLE=$(echo "$gh_pr_list_out" | cut -f 2)
```